### DISCUSSION TITLE
How to make window take up all available screen size

### DISCUSSION THREAD

**Discussion by nikita-skobov** [emilk/egui#225] [Q&A] [answered]
Hi,

Im able to query the screen size and then pass that in to:

```
ui.set_width(size.0);
ui.set_max_height(size.1);
ui.set_min_height(size.1);
```

And that achieves the result that I want: the window takes up the entire size of the screen
But is there an easier way to do that where I dont need to look up the screen size?

**Comment by emilk** [comment, marked as answer]
If you want a `Ui` that covers the screen, use [`CentralPanel`](https://docs.rs/egui/0.10.0/egui/containers/struct.CentralPanel.html) ?

**Reply by nikita-skobov** [reply]
yep thats it, perfect thanks

**Reply by mankinskin** [reply]
How do i do this when I have other panels too? Is there a global Ui?

**Comment by moon989** [comment]
**if you got the screen size , you can do as bellow . but could you tell me  how to query query the screen size**
`
fn main() {
    let app = AppFrameWrok::default();
    
    // transparent: Let's show off that we support transparent windows
    let native_options = eframe::NativeOptions{transparent: true,
        // set window size ; 
        initial_window_size:Some(eframe::egui::Vec2::new(1920.0, 1080.0)),
        ..Default::default()};    
    eframe::run_native(Box::new(app), native_options);
}

`
